Hindawi Publishers.js

```diff
@@ -80,7 +89,27 @@ function doWeb(doc,url)
 				// use article ID as a page (seems to be the last part of URL/DOI)
 				item.pages = 'e' + item.DOI.substr(item.DOI.lastIndexOf('/') + 1);
 			}
-			item.extra = "";
+			//remove duplicate metadata			
```

Comment:
I'm not really clear on what you're trying to remove from which fields. How is this duplicate metadata getting in. Is this only touching PMID in the extra field?